### PR TITLE
Change host port to 8080

### PR DESCRIPTION
**Pull request by cadef**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/cadef/python3-vagrant/pull/1" target="_blank">ReviewNinja

File: Vagrantfile [hunk 1]
```diff
@@ -4,7 +4,7 @@
 Vagrant.configure(2) do |config|
   config.vm.box = "ubuntu/trusty64"
   config.vm.provision :shell, path: "bootstrap.sh"
-  config.vm.network :forwarded_port, guest: 8000, host: 8080
+  config.vm.network :forwarded_port, guest: 8080, host: 8080
```
Comment (cadef) [inline Vagrantfile:7]: !fix Test comment

Comment (cadef) [inline Vagrantfile:7]: !fixed Fixed

Comment (cadef) [inline Vagrantfile:7]: Test comment 2

**Pull request merged**